.github/workflows/cd.yml

```diff
@@ -412,3 +345,5 @@ jobs:
           path: |
             changed.txt
             versions.json
+            ./packages/vscode-extension/*.vsix
+            ./packages/server/lib/*.exe
```

Comment:
The built server under "./packages/server/lib" has the version info as below: add server version as output to VS or change the "./packages/server/buildExe.js" to let the output named "server.exe".

![image](https://user-images.githubusercontent.com/15262146/222374821-31a9c08d-08e5-4651-931d-ea6a69b93ce8.png)